config/grunt/custom-tasks/init-rc-release.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+/**
+ * helper function
+ * @param {array} 
+ * @returns {bool}
+ */
+
+if (!String.prototype.isInList) {
+	String.prototype.isInList = function() {
+	   let value = this.valueOf();
+	   for (let i = 0, l = arguments.length; i < l; i += 1) {
+		  if (arguments[i] === value) return true;
+	   }
+	   return false;
+	}
+ }
+
+
+
+
+
+module.exports = function( grunt ) {
+	grunt.registerMultiTask(
+		"init-rc-release",
+		"central entry point to set options in one place",
+		function() {
+            const options = this.options({
+                failOnError: true,
+                enableSvnDeploy: false,
+                githubOwner: '',
+                githubRepo: '',
+                alternativeBranch: 'CI-test',
+                alternativeBranchPush: false,
+                enableGithubRelease: false,
+                noBump: false,
+        });
+        
+
+        let owner;
+        let repo;
+        const pluginVersionArg = grunt.option( "plugin-version" );
+        const type = grunt.option( 'type' ) || options.type;
+        const enableSvnDeployArg = grunt.option( 'enableSvnDeploy' ) ;
```

Comment:
These lines could use a helper function that takes the `option name` as argument and the option as second argument to set whenever the `option name` is not defined as grunt option.

Reply:
Good Idea, will try to come up with the necessary code, (but at the moment I have no idea how to do that in javascript ;-)

Reply:
```
function gruntOptionLogicToPreseveFalseValue( gruntOptionName, fallbackValue ) {
	const gruntOptionValue = grunt.option( gruntOptionName ) ;
    return (typeof gruntOptionValue === 'undefined' ) ? fallbackValue: gruntOptionValue;
}

```
I thank this should do the trick.